infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb

```diff
@@ -7,7 +7,7 @@
 # All rights reserved - Do Not Redistribute
 #
 
-cookbook_file "/home/akka/startup-script-cassandra-tweaks.sh" do
+cookbook_file "/home/ubuntu/startup-script-cassandra-tweaks.sh" do
```

Comment:
I didn't run the akka stuff on the c* nodes so moved the staging area to the ubuntu home dir

Reply:
ok 👍